src/util/misc.ts

```diff
@@ -171,21 +177,37 @@ import { cos } from './cos';
     getBisector: function (A, B, C) {
       var AB = fabric.util.createVector(A, B), AC = fabric.util.createVector(A, C);
       var alpha = fabric.util.calcAngleBetweenVectors(AB, AC);
-      //  check if alpha is relative to AB->BC
-      var ro = fabric.util.calcAngleBetweenVectors(fabric.util.rotateVector(AB, alpha), AC);
-      var phi = alpha * (ro === 0 ? 1 : -1) / 2;
       return {
-        vector: fabric.util.getHatVector(fabric.util.rotateVector(AB, phi)),
-        angle: alpha
+        vector: fabric.util.getHatVector(fabric.util.rotateVector(AB, alpha / 2)),
+        angle: Math.abs(alpha)
```

Comment:
For the sake of the pure method, not taking into consideration how we use it, I think we shouldn't return an abs angle. 
Am I missing something?
Why is it abs? I saw where it is used, maybe we should abs it there.
Thoughts?